src/contexts/ConfigurationContext.tsx

```diff
@@ -516,6 +520,7 @@ export const ConfigContextProvider = ({ children }: Props) => {
       ),
       summaryNumResults: config_summary_num_results ?? 7,
       summaryNumSentences: config_summary_num_sentences ?? 3,
+      summaryPromptOptions: config_summary_prompt_options?.split(","),
       summaryPromptName:
         config_summary_prompt_name ?? "vectara-experimental-summary-ext-2023-12-11-sml",
```

Comment:
Let's change the default here (if unspecified) to "vectara-summary-ext-24-05-sml"

Reply:
By Default it would be using the prompt assigned to `summaryPromptName `. In the option drawer we'll have option to change it.  Do you think we should change the default for the summaryPromptName? 

I did't removed the summaryPromptName for backword compatibility.